tsnet/tsnet.go

```diff
@@ -139,6 +144,63 @@ func (s *Server) LocalClient() (*tailscale.LocalClient, error) {
 	return s.localClient, nil
 }
 
+// LocalAPI returns a loopback ip:port listening for the "LocalAPI".
+//
+// As the LocalAPI is powerful, access to endpoints requires passing a
+// "Sec-Tailscale-LocalAPI" HTTP header containing the value of cred.
+//
+// It will start the server and the local client listener if they have not
+// been started yet.
+//
+// If you only need to use the LocalAPI from Go, then prefer LocalClient
+// as it does not require communication via TCP.
+func (s *Server) LocalAPI() (addr string, cred string, err error) {
```

Comment:
"LocalAPI" seems too similar to "LocalClient" without an obvious explanatory role. Maybe something that describes the return value, e.g., "LocalAPIAddress"?
```suggestion
func (s *Server) LocalAPI() (addr, cred string, err error) {
```